<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
    "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" >
    <head>
        <meta http-equiv="content-type" content="text/html; charset=iso-8859-1" />
        <meta name="author" content="Lior Pachter" />
        <meta name="keywords" content="Computational biology" />
        <meta name="description" content="" />
        <meta name="robots" content="all" />
        <title>Pachter Lab</title>
        <style type="text/css" media="all">
            @import "style.css";
        </style>
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">

    </head>
    <body>
      <div id="container">
          <div class="Header">
            <div id="logos">
              <a href="http://www.caltech.edu/"><img src="Caltech_LOGO-WHITE-RGB.png" alt = "Caltech logo" width=150 /></a>
              <p><span>Pachter Lab</span></p>
            </div>
              <div id="Menu">
                 <ul id="links">
                     <li><a href="index.html" title="Home">Home</a></li>
                     <li><a href="group.html" title="Group">Group</a></li>
                     <li><a href="research.html" title="Research" >Research</a></li>
                     <li><a href="software.html" title="Software" class="current-page">Software</a></li>
                     <li><a href="positions.html" title="Positions">Positions</a></li>
                     <li><a href="contact.html" title="Contact">Contact</a></li>
                     <li><a href="https://scholar.google.com/citations?user=DKfEcuEAAAAJ&hl=en&oi=ao" title="Publications">Publications <i class="fa fa-external-link" style="font-size:10px"></i> </a></li>
                     <li><a href="http://liorpachter.wordpress.com/" title="Blog">Blog <i class="fa fa-external-link" style="font-size:10px"></i> </a></li>
                 </ul>
             </div>
             <!-- end Menu-->
            </div>
            <!-- end Header-->
            <div id="Content">
                <dl>
                    <dt>Software developed in the Pachter group and still under active development in the group</dt>
                    <ul>
                        <li>Software and scripts required to reproduce results of papers are hosted at the <a href="https://github.com/pachterlab">Pachter Lab GitHub Repository</a>.</li>
                    </ul>
                </dl>
                <dl>
                    <dt>Software developed in the Pachter group but now maintained/developed elsewhere</dt>
                    <ul>
                        <li><a href="http://atifrahman.github.io/SWALO/">SWALO</a> (2016) Scaffolding with assembly likelihood optimization (now supported by Atif Rahman)</li>
                        <li><a href="http://pachterlab.github.io/PROBer/">PROBer</a> (2015) Quantitative modeling of transcriptome-wide RNA structure-probing experiments (now supported by Bo Li)</li>
                        <li><a href="http://pachterlab.github.io/cgal/">CGAL</a> (2013) Computing genome assembly likelihoods (now supported by Atif Rahman)</li>
                        <li><a href="http://fsa.sourceforge.net">FSA</a> (2009) <b>F</b>ast <b>S</b>tatistical <b>A</b>lignment (now supported by <a href="http://labs.fhcrc.org/bradleyr/">Robert Bradley</a> and his group at FHCRC)</li>
                        <li><a href="http://www.biostat.wisc.edu/~cdewey/mercator/">MERCATOR</a> (2004) Homology mapping (now supported by <a href="http://www.biostat.wisc.edu/~cdewey/">Colin Dewey</a> and his group at the University of Wisconsin)</li>
                        <li><a href="http://genome.lbl.gov/vista/mvista/about.shtml">VISTA</a> (2000) Visualization tool for global alignments
                            (now supported by <a href="http://www.jgi.doe.gov/whoweare/dubchak.html">Inna Dubchak</a> and her group  at the JGI)
                        </li>
                    </ul>
                </dl>
                <dl>
                    <dt>Retired Software</dt>
                    These programs, originally developed in the Pachter group, are no longer under active development and are not being supported.
                    <ul>
                        <li><b>eXpress</b> (2012) Streaming quantification for high-throughput sequencing (recommended instead: <a href="http://pachterlab.github.io/kallisto/">kallisto</a>)</li>
                        <li><b>SysCall</b> (2011) Distinguishing heterozygous sites from systematic error in high-thoughput sequenced reads (archived <a href="http://pachterlab.github.io/SysCall/">here</a>)</li>
                        <li><b>ReadSpy</b> (2012) Assessment of uniformity in RNA-Seq reads</li>
                        <li><b>MetMap</b> (2010) Analysis of Methyl-Seq experiments</li>
                        <li><b>Cufflinks</b> (2010) Transcript assembly and abundance estimation for RNA-Seq (recommended instead: <a href="http://pachterlab.github.io/kallisto/">kallisto</a> and <a href="http://pachterlab.github.io/sleuth/">sleuth</a>)</li>
                        <li><b>Tophat</b> (2009) Splice junction mapper for short RNA-seq reads (archived <a href="https://ccb.jhu.edu/software/tophat/index.shtml">here</a>)</li>
                        <li><b>AMAP</b> (2007) Protein multiple alignment </li>
                        <li><b>GENEMAPPER</b> (2006) Reference based gene annotation (recommended instead: <a href="http://www.nature.com/nbt/journal/v28/n5/abs/nbt.1621.html"> an RNA-Seq experiment</a>)</li>
                        <li><b>MJOIN</b> (2006) Neighbor joining with subtree weights </li>
                        <li><b>PARALIGN</b> (2006) Alignment polytope construction </li>
                        <li><b>SLIM</b> (2003) Minimum network design for optimizing the search space for pair hidden Markov models </li>
                        <li><b>SLAM</b> (2003) Pairwise simultaneous alignment and gene finding (recommended instead: <a href="http://www.nature.com/nbt/journal/v28/n5/abs/nbt.1621.html">an RNA-Seq experiment</a>)</li>
                        <li><b>MAVID</b> (2003) Multiple alignment of large genomic sequences </li>
                    </ul>
                </dl>
            </div>
        </div>
    </body>
</html>
